app/src/main/java/org/schabi/newpipe/local/playlist/LocalPlaylistManager.java

```diff
@@ -82,6 +86,51 @@ public Completable updateJoin(final long playlistId, final List<Long> streamIds)
         })).subscribeOn(Schedulers.io());
     }
 
+    public Maybe<List<PlaylistStreamEntry>> clearWatched(final long playlistId,
+                                                         final boolean remPartiallyWatched) {
```

Comment:
Do not truncate names
```suggestion
                                                         final boolean removePartiallyWatched) {
```

Reply:
Whoops, forgot to change it, I'll reset the commit now und change it correctly ;)